src/Breweries.php

```diff
@@ -4,20 +4,38 @@
 
 namespace Interns2022B;
 
+use Illuminate\Support\Collection;
+use Interns2022B\Models\Brewery;
+
 class Breweries
 {
     public const SEARCH = "choice";
 
-    public function getBreweries(array $breweriesData): array
+    public function getBreweries(Collection $breweriesFactory): Collection
     {
         $name = readline("Please provide brewery or city name:");
 
-        $chosenData = [];
-        foreach ($breweriesData as $rowValue) {
-            if (in_array($name, $rowValue, strict: true)) {
-                $chosenData[] = $rowValue;
+        $chosenData = new Collection();
+        $chosenData->push(["Brewery name", "City", "Country", "Provider"]);
```

Comment:
Remove new Collection and use make method
```php
$chosenData = Collection::make(["Brewery name", "City", "Country", "Provider"]);
```

Reply:
Ola sent a screenshot of the problem on slack

Reply:
So keep it as it is